draft-ietf-tls-tls13.md

```diff
@@ -5485,13 +5506,60 @@ compute all future traffic secrets on that connection.  Systems which want such
 guarantees need to do a fresh handshake and establish a new connection with an
 (EC)DHE exchange.
 
-
 ### External References
 
 The reader should refer to the following references for analysis of the TLS record layer:
 {{BMMT15}} {{BT16}} {{BDFKPPRSZZ16}} {{BBK17}}.
 
 
+## Traffic Analysis
+
+TLS is susceptible to a variety of traffic analysis attacks based on
+observing the length and timing of encrypted packets {{MHJT14}}
+{{HCJC16}}.  This is particularly easy to see when there is a small
+set of possible messages to be distinguished, such as for a video
+server hosting a fixed corpus of content, but still provides usable
+information even in more complicated scenarios.
+
+TLS does not provide any specific defenses against this form of attack
+but does include a padding mechanism for use by applications: The
+plaintext protected by the AEAD function consists of content plus
+variable-length padding, which allows the application to produce
+arbitrary length packets as well as cover traffic.  Because the
```

Comment:
I think that "cover traffic" needs definition.  That is, traffic that contains no application data, but exists to reduce the attacker's ability to infer the properties of records that do contain data, or something like that.